presto-hive/src/main/java/com/facebook/presto/hive/HivePageSourceProvider.java

```diff
@@ -765,7 +765,7 @@ public int getIndex()
         }
 
         /**
-         * @param columns columns that need to be returned to engine
+         * @param columns that need to be returned to engine
```

Comment:
Oops, while I'd tuned it not to flag this case, it appears that I did apply this change. I'll drop.